lib/ansible/modules/source_control/github_deploy_key.py

```diff
@@ -241,6 +246,7 @@ def main():
 
     module = AnsibleModule(
         argument_spec=dict(
+            github_url=key=dict(required=False, type='str', default="https://api.github.com"),
```

Comment:
```suggestion
            github_url=dict(default="https://api.github.com"),
```

Reply:
`key=` is not required. `required=False` and `type='str'` are defaults so no need to mention.

Reply:
Fixed